app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-group">
+  <%= form_for (@my_post) do |f| %>
```

Comment:
There is a bug here when I try to update the post with empty content. It is always good practice to test the forms with empty fields. Check the error here #4.

Reply:
I've fixed the update bug.